src/api/flaskr/service/learn/context_v2.py

```
@@ -3140,19 +3140,36 @@ def _process_stream_chunk(
                         return
-                    try:
-                        yield from tts_processor.drain_ready_segments()
-                    except Exception as exc:
-                        app.logger.warning(
-                            "Idle streaming TTS drain failed; disable for this block: %s",
-                            exc,
-                            exc_info=True,
+                    for processor in processors:
+                        try:
+                            yield from processor.drain_ready_segments()
+                        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)